spec/surfnperf_spec.js

```diff
@@ -1096,5 +1096,74 @@ define('spec/surfnperf_spec', [
 
     });
 
+
+    describe('#chromeLoadTimes', function() {
+      it('returns a reference to window.chrome.loadTimes() if the browser supports it, otherwise undefined', function() {
+        if(window.chrome && window.chrome.loadTimes()) {
+          expect(SurfNPerf.chromeLoadTimes()).toEqual(window.chrome.loadTimes());
+        } else {
+          expect(SurfNPerf.chromeLoadTimes()).toEqual(undefined);
+        }
+      });
+    });
+
+    describe('#msFirstPaint', function() {
+      it('returns a reference to window.performance.timing.msFirstPaint if the browser supports it, otherwise undefined', function() {
+        if(window.performance.timing.msFirstPaint) {
+          expect(SurfNPerf.msFirstPaint()).toEqual(window.performance.timing.msFirstPaint);
+        } else {
+          expect(SurfNPerf.msFirstPaint()).toEqual(undefined);
+        }
+      });
+    });
+
+    describe('#navigationStartTiming', function() {
+      it('returns a reference to window.performance.timing.navigationStart() if the browser supports it, otherwise undefined', function() {
+        if(window.performance && window.performance.timing && window.performance.timing.navigationStart) {
+          expect(SurfNPerf.navigationStartTiming()).toEqual(window.performance.timing.navigationStart());
+        } else {
+          expect(SurfNPerf.navigationStartTiming()).toEqual(undefined);
+        }
+      });
+    });
+
+    describe('#getFirstPaint', function() {
+      describe('when the client is using Chrome', function() {
+        it('returns the time to first paint relative to navigation start time', function() {
+          spyOn(SurfNPerf, 'chromeLoadTimes').andReturn({
+            firstPaintTime: 1446480965.123456
+          });
+          spyOn(SurfNPerf, 'msFirstPaint').andReturn(undefined);
+          spyOn(SurfNPerf, 'navigationStartTiming').andReturn(123789.734);
+
+          expect(SurfNPerf.getFirstPaint()).toEqual(1446480965.123456 * 1000 - SurfNPerf.navigationStartTiming());
```

Comment:
and then here the `toEqual` would be `1446480965.123456 * 1000 - SurfNPerf.performanceTiming().navigationStart`